### ISSUE TITLE
Make a plan for data validation

### ISSUE THREAD

**Issue by jsignell** [NASA-IMPACT/veda-data-pipelines#340]
## PI Objective
- https://github.com/NASA-IMPACT/veda-architecture/issues/222
- Maintenance/operations

## Description
Make a plan before starting work, in particular

1. Determine when and how data provider will run validation
2. Describe at which stage the data provider should hand off to the development team
3. Clarify how this work relates to priors such as rio-stac, xstac, stactools, and stactools-packages

## Acceptance Criteria
- [x] planning session with @jsignell and @ividito to scope work and coordinate development tasks
- [ ] documentation of decisions and user stories

**Comment by jsignell** [comment]
Just to bring this full circle here is the link to the document that we are working off of: https://docs.google.com/document/d/1p5w5uMJwZFGdNKqLnA5BkP2b5n1yjdBtkkVbJ8bE1xo/edit#

**Comment by jsignell** [comment]
I think we have split this up into more actionable tickets.
